src/scripts.js

```diff
@@ -230,42 +283,41 @@ function flightsTodayElement() {
 }
 
 function stairsTodayElement() {
-  console.log('hello')
-  let stairsToday = user.activityData.find(activity => {
-    return activity.date === todayDate;
+  let stairsToday = activityData.find(activity => {
+    activity.userID === user.id && activity.date === todayDate;
+    return activity;
   })
   stairsUserStairsToday.innerText = stairsToday.flightsOfStairs * 12;
 }
 
 function stepElements() {
-  const calculatedMiles = 
-  user.activityData.find(activity => activity.date === todayDate);
-  stepsInfoMilesWalkedToday.innerText = calculatedMiles.calculateMiles() 
-  stepsCalendarTotalActiveMinutesWeekly.innerText = user.calculateAverageThisWeek(todayDate, 'activityData', 'minutesActive', 0);
-  stepsCalendarTotalStepsWeekly.innerText = user.calculateAverageThisWeek(todayDate, 'activityData', 'steps', 0);
+  stepsInfoMilesWalkedToday.innerText = user.activityRecord.find(activity => {
+    return (activity.date === todayDate && activity.userId === user.id)
+  }).calculateMiles(userRepository);
+  stepsCalendarTotalActiveMinutesWeekly.innerText = user.calculateAverageMinutesActiveThisWeek(todayDate);
```

Comment:
I think this may need a git pull to bring up to date, We have been switching over to calculateAverageThisWeek